### DISCUSSION TITLE
关于托盘集成的报错求助，getlantern/systray

### DISCUSSION THREAD

**Discussion by zk3151463** [wailsapp/wails#2690] [Q&A]
集成github.com/getlantern/systray时，运行报下面错误，单独运行托盘代码时没有问题，同样代码在win虚拟机里是好的，但是在mac(m1)报下面错误

/opt/homebrew/Cellar/go/1.20.2/libexec/pkg/tool/darwin_arm64/link: running cc failed: exit status 1
duplicate symbol '_OBJC_CLASS_$_AppDelegate' in:
    /var/folders/21/qhqcdppj2rn_v1dx_t64s9vh0000gn/T/go-link-3028710247/000003.o
    /var/folders/21/qhqcdppj2rn_v1dx_t64s9vh0000gn/T/go-link-3028710247/000026.o
duplicate symbol '_OBJC_METACLASS_$_AppDelegate' in:
    /var/folders/21/qhqcdppj2rn_v1dx_t64s9vh0000gn/T/go-link-3028710247/000003.o
    /var/folders/21/qhqcdppj2rn_v1dx_t64s9vh0000gn/T/go-link-3028710247/000026.o
ld: 2 duplicate symbols for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

Build error - exit status 1
No version running, build will be retriggered as soon as changes have been detected

**Comment by marcio199226** [comment]
try this one https://github.com/fyne-io/systray
